src/js/engagement_view/src/components/engagementView/dynamicEngagementViewFeatures/utils/toggleNodeTable.tsx

```diff
@@ -0,0 +1,45 @@
+import React, {useState} from "react";
+
+import Button from "@material-ui/core/Button";
+import LensIcon from '@material-ui/icons/Lens';
+import ExpandMoreIcon from '@material-ui/icons/ExpandMore';
+
+import {
+    ToggleNodeTableProps,
+} from "types/DynamicEngagementViewTypes";
+
+
+import {NodeDetails} from '../DynamicEngagementViewFeatures'
+
+import { useStyles } from '../styles';
+
+
+export function ToggleNodeTable({curNode}: ToggleNodeTableProps) {
+    const [toggled, toggle] = useState(true);
+    const classes = useStyles();
+    return (
+        <>
+        <div>
+            <div className={classes.header}>
+                <b className={classes.title}><LensIcon className={classes.icon}/> NODE</b>
+                <Button
+                    className = {classes.button}
+                    onClick={
+                        () => { toggle(toggled => !toggled) }
+                    }> 	
+                    <ExpandMoreIcon className={classes.expand}/> 
+                </Button>
+            </div>
+
+            <div className="nodeToggle">
+                {
+                    toggled && curNode && 
+                        <>
```

Comment:
these empty tags are new to me, what's the purpose? does it just say "hey we're doing jsx now"?

Reply:
ya, they're like delimiters for inline HTML:  https://reactjs.org/docs/fragments.html#short-syntax

Reply:
This is a bad habit of mine, JSX requires parent elements, so I usually start with <></> but don't always need them. 

Removed. Nice Catch, thanks!